.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -2,6 +2,19 @@ name: Report a Bug
 description: Use this template to report a Pymatgen-related bug
 labels: bug
 body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Try the new Codespaces, setup env and collect info made easy.
+        <details>
+          <summary>🚀 Click to Show the Codespaces Guide</summary>
+          <img width="300" alt="Create codespace" src="https://github.com/materialsproject/pymatgen/assets/80093591/482181f8-d7e1-49b4-8382-2d948ed4e550">
+
+          1. Open up the [pymatgen repo](https://github.com/materialsproject/pymatgen) in your web browser.
+          2. Click the green `Code` button at the top-right corner of the repo, and switch to the `Codespaces` panel.
+          3. Click `Create code space on master` to create a new codespace, or to return to an existing codespace.
+        </details>

```

Comment:
Rather than this multistep process, can we not just show an SVG badge with a deeplink to open a new codespace? see [Open in GitHub Codespaces docs](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/setting-up-your-repository/facilitating-quick-creation-and-resumption-of-codespaces#creating-an-open-in-github-codespaces-badge)

Reply:
@janosh Wonderful suggestion! Thanks and implemented.